tpp-prophets/peptide_prophet.xml

```diff
@@ -18,92 +17,66 @@
             $accurate_mass
             $no_ntt
             $no_nmc
-            $use_gamma
+            $usegamma
             $use_only_expect
             $force_fit
             $allow_alt_instruments
             $maldi
             $usedecoys
-            --decoy-prefix $decoy_prefix_string
+            --decoy-prefix '$decoy_prefix_string'
 
             #if str($experiment_label).strip():
                 --experiment-label '$experiment_label'
             #end if
-
+            --database '$database'
```

Comment:
This was needed as otherwise Galaxy was searching for a database table in `/Users/icooke/Desktop/iptest/AASequences.fasta`